src/LoginControl.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react"
 import Button from "react-bootstrap/Button"
 
 interface LoginControlProps {
-  loggedIn: boolean
+  loggedIn: string
```

Comment:
I don't think this should change. This component isn't concerned with what the username is. It's only concerned with a binary "user is logged in" or "user is not logged in."

Reply:
Okk ! Understood ! I will only send the result of the state over here ! Rather than sending the userId itself.